pkg/manifests/config.go

```diff
@@ -185,7 +185,7 @@ func (c *Config) applyDefaults() {
 func (c *Config) SetImages(images map[string]string) {
 	c.Images.PrometheusOperator = images["prometheus-operator"]
 	c.Images.PrometheusConfigReloader = images["prometheus-config-reloader"]
-	c.Images.ConfigmapReload = images["configmap-reload"]
+	c.Images.ConfigmapReload = images["configmap-reloader"]
```

Comment:
super small nit: let's rename the var to `c.Images.ConfigmapReloader` too, in order to have it consistent

Reply:
ok